src/App.tsx

```diff
@@ -1,14 +1,33 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { GlobalContext } from './reducer';
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [state, dispatch] = useContext(GlobalContext);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(response => {
+        dispatch({ type: 'RequestListTodos', list: response });
+        setIsLoading(false);
+      })
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.warn('Error:', error);
+        setIsLoading(false);
+      });
+  }, []);
```

Comment:
do you hear about finally?
```suggestion
    getTodos()
      .then(response => {
        dispatch({ type: 'RequestListTodos', list: response });
      })
      .catch(error => {
        // eslint-disable-next-line no-console
        console.warn('Error:', error);
      })
    .finally(() => setIsLoading(false));
  }, []);
```